scrapers/EliteBabes.yml

```diff
@@ -0,0 +1,50 @@
+name: EliteBabes
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.elitebabes.com/?s={}
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - www.elitebabes.com/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  performerSearch:
+    performer:
+      Name: //article[@id='content']/ul[@class='gallery-a b d']/li/a[contains(@href,"model/")]/span/text()
```

Comment:
```suggestion
      Name: //article[@id='content']/ul[1]//span/text()
```

Slightly different strategy here that makes this much more compact. I select the first `ul` node, since all the other `ul` nodes on the page share the same class name anyway, from there we no longer need the contains and can go straight down to the first span node containing text

Reply:
I'm not sure this is correct. In the case of a search that results in no performer found but only gallery, this would pull the gallery instead...

Reply:
Good point, this one works instead:
```suggestion
      Name: //article[@id='content']/*[following-sibling::h2]//span/text()
```
Selects all the elements who's following sibling is an h2 under content. That way galleries will always be filtered out.

Reply:
As mentioned on discord, i disagree with this edits. It make the scraper more brittle for no reason.

Reply:
```suggestion
      Name: //ul//a[contains(@href,"model/")]//text()
```

Super short version that leaves the query in and strips out the unneeded parts of the path